gpsd_client/config/gpsd_client.yaml

```diff
@@ -1,6 +1,6 @@
 gpsd_client:
   ros__parameters:
-    use_gps_time: false
-    check_fix_by_variance: false
+    use_gps_time: true
+    check_fix_by_variance: true
```

Comment:
I'm not a big fan of changing the default values of these in with this PR. I generally agree that these are probably better defaults, but I'd rather not surprise people with this change.

Reply:
Good point. I've pulled that commit back out.